htlcswitch/link_test.go

```diff
@@ -39,6 +39,7 @@ import (
 
 const (
 	testStartingHeight = 100
+	testDefaultDelta   = 6
```

Comment:
For commits that only change test code, we often use `htlcswitch/test` as the commit message prefix